pipeline/03-evaluate.R

```diff
@@ -150,7 +156,7 @@ gen_agg_stats <- function(data, truth, estimate, bldg_sqft,
 
       # Assessment-specific ratio stats
       rs_lst = rs_fns_list %>%
-        map(., \(f) exec(f, pmin({{ estimate }}, 1), {{ truth }})) %>%
+        map(., \(f) exec(f, pmax({{ estimate }}, 1), {{ truth }})) %>%
```

Comment:
Accidentally setting all the assessed values to 1 when calculating ratio stats is a fun time.